front/src/applications/stdcm/types.ts

```diff
@@ -200,3 +200,8 @@ export type StdcmLinkedPathResult = {
 };
 
 export type ExtremityPathStepType = 'origin' | 'destination';
+
+export type LoaderStatus = {
+  status: 'fixed' | 'sticky' | 'absolute';
```

Comment:
Maybe we can use `fixed-top` and `fixed-bottom` instead of `fixed` and `sticky`
```suggestion
  status: 'fixed-top' | 'fixed-bottom' | 'absolute';
```

Reply:
Done